jest.config.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  preset: "ts-jest/presets/js-with-ts",
+  testMatch: [
+    "<rootDir>/packages/*/src/**/__tests__/**/*.{js,jsx,ts,tsx}",
+    "<rootDir>/packages/*/src/**/?(*.)(spec|test).{js,jsx,ts,tsx}",
+  ],
+  coveragePathIgnorePatterns: [
+    "/node_modules/",
+    "dist/",
+    "<rootDir>/packages/components/src/index.ts",
```

Comment:
Why is this needed if it doesn't match the spec?

Reply:
this is for code coverage metrics. not sure why this file was specifically excluded before, but i kept it here to not break anything historical